### PR TITLE
prevent submiting of form when typeahead container is visible

### PR DESCRIPTION
**Pull request by fkowal**
gh-359

### CODE REVIEW

File: components/typeahead/typeahead.directive.ts [hunk 1]
```diff
@@ -220,6 +220,16 @@ export class Typeahead implements OnInit {
     }
   }
 
+  @HostListener('keydown', ['$event'])
+  onKeydown(e:KeyboardEvent) {
+    // When typeahead container is visible, prevent submitting the form
+    if (this.container) {
```
Comment (valorkin) [inline components/typeahead/typeahead.directive.ts]: please reduce nesting
`if (this.container && e.keyCode === 13)`
should be enough

**Comment by valorkin** [comment]
looks good! one things, please squash to one commit and rename it to:
`fix(typeahead): prevent form submition when typeahead selected (fixes #359)`

**Comment by fkowal** [comment]
done

**Comment by valorkin** [comment]
cool! merging

**Comment by fkowal** [comment]
what does the beta.14 translate to ?

i am basically asking about a release date

**Comment by valorkin** [comment]
just a list of issues before next ng2 release
if you want I can publish patch with this

**Comment by valorkin** [comment]
just don't want to over use https://cdnjs.com/libraries/ng2-bootstrap

**Comment by fkowal** [comment]
I'll Ping in s bit if it will raje to long

**Comment by valorkin** [comment]
ok, you can use latest ng2-bootstrap directly from master
if any

**Pull request merged**